### DISCUSSION TITLE
[Feature] Allow filtering on LinkToAnot... [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1923] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### This is available since 0.90 version  
#  
> 💁 **o1lab**  
#### as pointed by one of our user, you can only one linkToAnotherRecord in the filter and ability to improve it is been tracked in github  
#  
> 🙋‍♂️ **Nils**  
#### ahhh, nice!  
#  
> 🙋‍♂️ **Nils**  
#### I was running NocoDB with version `0.84.1`, because I just cloned `nocodb-seed` repo and didn't upgraded the NocoDB package  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966396726212702278)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
